1. pam_systemd doesn't exist on void, use pam_elogind instead and make pam_systemd optional
2. pam_turnstile allows turnstile to integrate with lightdm

--- a/data/pam/lightdm
+++ b/data/pam/lightdm
@@ -17,7 +17,9 @@ password  required pam_unix.so
 
 # Setup session
 session   required pam_unix.so
-session   optional pam_systemd.so
+-session   optional pam_turnstile.so
+-session   optional pam_elogind.so
+-session   optional pam_systemd.so
 
 # Load limits
 session   optional pam_limits.so

--- a/data/pam/lightdm-autologin
+++ b/data/pam/lightdm-autologin
@@ -19,7 +19,9 @@ password  required pam_deny.so
 
 # Setup session
 session   required pam_unix.so
-session   optional pam_systemd.so
+-session   optional pam_turnstile.so
+-session   optional pam_elogind.so
+-session   optional pam_systemd.so
 
 # Load limits
 session   optional pam_limits.so

--- a/data/pam/lightdm-greeter
+++ b/data/pam/lightdm-greeter
@@ -14,7 +14,9 @@ password  required pam_deny.so
 
 # Setup session
 session   required pam_unix.so
-session   optional pam_systemd.so
+-session   optional pam_turnstile.so
+-session   optional pam_elogind.so
+-session   optional pam_systemd.so
 
 # Load limits
 session   optional pam_limits.so
